shared/actions/tracker2.tsx

```diff
@@ -279,7 +303,8 @@ function* tracker2Saga() {
   yield* Saga.chainAction(Tracker2Gen.changeFollow, changeFollow)
   yield* Saga.chainAction(Tracker2Gen.ignore, ignore)
   yield* Saga.chainGenerator<Tracker2Gen.LoadPayload>(Tracker2Gen.load, load)
-  yield* Saga.chainAction(Tracker2Gen.load, loadFollow)
+  yield* Saga.chainAction(Tracker2Gen.load, loadFollowers)
```

Comment:
is there some motivation for splitting it into 2 side effects now. before it had one side effect and made 2 calls and joined them. This could still do the same

Reply:
With my change, if you're loading someone with a large sigchain like chris for the first time, loading the following list takes a lot longer than the followers list. I split it up so the followers tab can populate first. I can change it if there's a better way to do the same thing